api/src/opentrons/protocol_api/labware.py

```diff
@@ -117,8 +117,19 @@ def parent(self) -> Labware:
     @property
     @requires_version(2, 0)
     def has_tip(self) -> bool:
-        """Whether this well contains a tip. Always ``False`` if the parent labware
-        isn't a tip rack."""
+        """Whether this well contains an unused tip.
+
+        From API v2.2 on:
+        - Returns ``False`` if:
+            - the well has no tip present or
+            - the well has a tip that's been used by the protocol previously.
+        - Returns ``True`` if the well has an unused tip.
+
+        Before API v2.2:
+        - Returns ``True`` as long as the well has a tip, even if it is used.
+
+        Always ``False`` if the parent labware isn't a tip rack.
+        """
```

Comment:
Looks like Sphinx is having trouble with the nested list.

This is ugly but I think it works:

```suggestion
        """Whether this well contains an unused tip.

        From API v2.2 on:

        - Returns ``False`` if:

          - the well has no tip present, or
          - the well has a tip that's been used by the protocol previously

        - Returns ``True`` if the well has an unused tip.

        Before API v2.2:

        - Returns ``True`` as long as the well has a tip, even if it is used.

        Always ``False`` if the parent labware isn't a tip rack.
        """
```